onnxruntime/python/tools/pytorch_export_contrib_ops.py

```diff
@@ -5,15 +5,16 @@
 Support for registering ONNX Runtime's built-in contrib ops with
 PyTorch-ONNX exporter (torch.onnx.export).
 """
-
 import typing
 
+import torch._C
 try:
     from torch.onnx import register_custom_op_symbolic
```

Comment:
move this outside the try: it should only wrap the first `torch` import.

Reply:
Done